petl/io/pytables.py

```diff
@@ -17,35 +17,36 @@ def fromhdf5(source, where=None, name=None, condition=None,
     Provides access to an HDF5 table. E.g.::
 
         >>> import petl as etl
-        >>> import tables
+        >>>
         >>> # set up a new hdf5 table to demonstrate with
-        ... h5file = tables.open_file('example.h5', mode='w',
-        ...                           title='Example file')
-        >>> h5file.create_group('/', 'testgroup', 'Test Group')
-        /testgroup (Group) 'Test Group'
-          children := []
-        >>> class FooBar(tables.IsDescription):
-        ...     foo = tables.Int32Col(pos=0)
-        ...     bar = tables.StringCol(6, pos=2)
-        ...
-        >>> h5table = h5file.create_table('/testgroup', 'testtable', FooBar,
-        ...                               'Test Table')
-        >>> # load some data into the table
-        ... table1 = (('foo', 'bar'),
-        ...           (1, b'asdfgh'),
-        ...           (2, b'qwerty'),
-        ...           (3, b'zxcvbn'))
-        >>> for row in table1[1:]:
-        ...     for i, f in enumerate(table1[0]):
-        ...         h5table.row[f] = row[i]
-        ...     h5table.row.append()
-        ...
-        >>> h5file.flush()
-        >>> h5file.close()
+        >>> class FooBar(tables.IsDescription): # doctest: +SKIP
+        ...     foo = tables.Int32Col(pos=0) # doctest: +SKIP
+        ...     bar = tables.StringCol(6, pos=2) # doctest: +SKIP
         >>> #
-        ... # now demonstrate use of fromhdf5
-        ... table1 = etl.fromhdf5('example.h5', '/testgroup', 'testtable')
-        >>> table1
+        >>> def setup_hdfs5_table():
```

Comment:
```suggestion
        >>> def setup_hdf5_table():
```

Avoid confusion with hdfs (something different).